test_conformance/opencl_conformance_tests_full_no_math_or_conversions.csv

```diff
@@ -39,23 +39,23 @@ Device Partitioning,device_partition/test_device_partition
 # Buffers and images
 # #########################################
 Buffers,buffers/test_buffers
-Images (API Info),images/clGetInfo/test_cl_get_info 
-Images (Kernel Methods),images/kernel_image_methods/test_kernel_image_methods 
+Images (API Info),images/clGetInfo/test_cl_get_info
+Images (Kernel Methods),images/kernel_image_methods/test_kernel_image_methods
 Images (Kernel),images/kernel_read_write/test_image_streams CL_FILTER_NEAREST
 Images (Kernel pitch),images/kernel_read_write/test_image_streams use_pitches CL_FILTER_NEAREST
 Images (Kernel max size),images/kernel_read_write/test_image_streams max_images CL_FILTER_NEAREST
 Images (clCopyImage),images/clCopyImage/test_cl_copy_images
 Images (clCopyImage small),images/clCopyImage/test_cl_copy_images small_images
-Images (clCopyImage max size),images/clCopyImage/test_cl_copy_images max_images 
-Images (clReadWriteImage),images/clReadWriteImage/test_cl_read_write_images 
-Images (clReadWriteImage pitch),images/clReadWriteImage/test_cl_read_write_images use_pitches 
-Images (clReadWriteImage max size),images/clReadWriteImage/test_cl_read_write_images max_images 
-Images (clFillImage),images/clFillImage/test_cl_fill_images 
-Images (clFillImage pitch),images/clFillImage/test_cl_fill_images use_pitches 
-Images (clFillImage max size),images/clFillImage/test_cl_fill_images max_images 
-Images (Samplerless),images/samplerlessReads/test_samplerless_reads 
-Images (Samplerless pitch),images/samplerlessReads/test_samplerless_reads use_pitches 
-Images (Samplerless max size),images/samplerlessReads/test_samplerless_reads max_images 
+Images (clCopyImage max size),images/clCopyImage/test_cl_copy_images max_images
+Images (clReadWriteImage),images/clReadWriteImage/test_cl_read_write_images
+Images (clReadWriteImage pitch),images/clReadWriteImage/test_cl_read_write_images use_pitches
+Images (clReadWriteImage max size),images/clReadWriteImage/test_cl_read_write_images max_images
+Images (clFillImage),images/clFillImage/test_cl_fill_images
+Images (clFillImage pitch),images/clFillImage/test_cl_fill_images use_pitches
+Images (clFillImage max size),images/clFillImage/test_cl_fill_images max_images
+Images (Samplerless),images/samplerlessReads/test_samplerless_reads
+Images (Samplerless pitch),images/samplerlessReads/test_samplerless_reads use_pitches
+Images (Samplerless max size),images/samplerlessReads/test_samplerless_reads max_images
```

Comment:
I can't comment on the exact line, but further down this file there is still an `-ILPath`.

Reply:
Also the renamed `opencl_conformance_tests_full_spirv.csv ` file but I can't even comment on that file nevermind the line!

Reply:
Done here: https://github.com/KhronosGroup/OpenCL-CTS/pull/981